### ISSUE TITLE
number of threads for EM simulation

### ISSUE THREAD

**Issue by iorga** [Becksteinlab/MDPOW#119]
Currently, the EM simulation is executed with the command `gmx mdrun -c em.pdb -deffnm em -stepout 10 -v` and no parameter can be changed by the user.

In my case, it works on a machine with 8 cores, but not on another machine with 20 cores, which gives the following error:
```
Will use 15 particle-particle and 5 PME only ranks
...
Fatal error:
There is no domain decomposition for 15 ranks that is compatible with the
given box and a minimum cell size of 0.449686 nm
Change the number of ranks or mdrun option -rdd or -dds
Look in the log file for details on the domain decomposition
```

We need that the user can change at least the number of threads for the EM simulation, otherwise MDPOW cannot be run on certain machines. @orbeckst @VOD555 can you have a look, please ?

**Comment by orbeckst** [comment]
@iorga @VOD555 I patched this functionality in in PR #128 but without tests. I am not happy about this but we need to fix the tests anyway and one user confirmed that the fixed worked. For right now, the mdrun.maxthreads setting will also apply to energy minimization.

**Comment by VOD555** [comment]
Ok. I think we need to open an issue about the tests needed to be added.
